subsys/bluetooth/host/keys.h

```diff
@@ -6,6 +6,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#ifndef __BT_HOST_KEYS_H
```

Comment:
we don't use include guards in zephyr

Reply:
We do, but generally not for internal header files, to catch circular dependencies more easily.